main_train_psnr.py

```diff
@@ -9,6 +10,9 @@
 from torch.utils.data import DataLoader
 from torch.utils.data.distributed import DistributedSampler
 import torch
+from clearml import Task
+from clearml import Dataset
+import os
```

Comment:
check import duplication and import sorting

using isort and including the config from ns agriculture